### PR TITLE
Script fixes

### PR DESCRIPTION
**Pull request by rafatower**
A bunch of small fixes for the scripts to get them working on my machine :tm:

@rochoa can you please review?

### CODE REVIEW

File: node-mapnik/scripts/test [hunk 1]
```diff
@@ -27,13 +27,13 @@ PGUSER=postgres psql -d template_postgis -c "GRANT ALL ON spatial_ref_sys TO PUB
...
 # Link previously built node-mapnik
 cd node-mapnik
-npm link
+sudo npm link
 cd ..
 
 # Clone windshaft, link mapnik to it and run tests
 git clone https://github.com/cartodb/windshaft
 cd windshaft
-npm install
```
Comment (rochoa) [inline node-mapnik/scripts/test:36]: I don't link this approach. I prefer to link an already `install`ed module.

File: node-mapnik/scripts/test [hunk 2]
```diff
@@ -27,13 +27,13 @@ PGUSER=postgres psql -d template_postgis -c "GRANT ALL ON spatial_ref_sys TO PUB
 
 # Link previously built node-mapnik
 cd node-mapnik
-npm link
+sudo npm link
```
Comment (rochoa) [inline node-mapnik/scripts/test]: I would avoid sudo. Options: https://docs.npmjs.com/getting-started/fixing-npm-permissions

Comment (rafatower) [inline node-mapnik/scripts/test]: this is intended to run in a vagrant machine. I'm aware of other approaches but the point here is to run this w/o manual intervention and it does NOT work as it is.

File: node-mapnik/scripts/test (hunk 1 above)
Comment (rafatower) [inline node-mapnik/scripts/test:36]: why?

Comment (rochoa) [inline node-mapnik/scripts/test:36]: Just an example: https://github.com/npm/npm/issues/10343

File: node-mapnik/scripts/test (hunk 2 above)
Comment (rochoa) [inline node-mapnik/scripts/test]: Then what about using:

```
NPM_CONFIG_PREFIX=~/.npm-global npm link
```

And the same later on when you need to link it as a dependency.

**Comment by rafatower** [comment]
is it now good to be merged?

**Pull request merged**